src/platforms/web/runtime/transition-util.js

```diff
@@ -179,3 +179,13 @@ function getTimeout (delays: Array<string>, durations: Array<string>): number {
 function toMs (s: string): number {
   return Number(s.slice(0, -1)) * 1000
 }
+
+export function parseDuration (value: Object): any {
```

Comment:
We can add the typing of number, NaN, or undefined to the return value, instead of `any`